cernopendata/modules/fixtures/data/docs/delphi-getting-started/delphi-getting-started.md

```diff
@@ -0,0 +1,439 @@
+[[_TOC_]]
+
+# Introduction
+This quick start guide is meant as a guide for the very first steps to get going with DELPHI software and data access.
+
+## Overview
+
+The /cvmfs/delphi.cern.ch repository contains the software binaries and sources for running:
+
+* DELPHI analysis framework (dstana)
+* MonteCarlo production (delsim/delana/shortdst production)
+* Event reconstruction (the DELPHI event server)
+* Event viewing with delgra
+
+## Before you start ...
+
+Please read and accept the data access rules.
+
+DELPHI data access rules are available at
+```
+https://delphi-www.web.cern.ch/delphi-www/delsec/finalrules/DELPHI_Data_preservation-8.pdf
+```
+Please read these before accessing the DELPHI software and data.
+
+Note that this software is provided "as is" within the DPHEP activity. Doing any analysis based on it requires careful cross checks with
+original data samples as well as blessing from the collaboration.
+
+## Requirements
+Binaries are provided for Linux only. There are no native Windows or Apple ports available, however, recent version of Windows offer the option to execute Linux binaries, e.g. for Ubuntu22, which has been tested and found to be working.
+
+To run the stack, you need to have access to a machine running one of the supported operating systems. A number of additional packages will be needed as well:
+* Please check https://gitlab.cern.ch/delphi/deployment for additional software packages which may be needed to for running and building your application. As an example, for Ubuntu these packages are needed:
+  * general: tcsh xfonts-100dpi xfonts-75dpi libxfont2
+  * compilers: cmake gcc g++ gfortran
+  * library packages: libx11 libglu1-mesa xutils libmotif r-base xutils libxbae libxaw7 libssl libglew libdlm
+* To mount /cvmfs, please check the instructions at https://cvmfs.readthedocs.io/en/stable/cpt-quickstart.html
+* To mount /eos, please check https://eoscta.docs.cern.ch/install/eos/ and https://eos-web.web.cern.ch/eos-web.
+
+These requirements are already satisfied on the interactive login clusters at CERN.
```

Comment:
The general public will not have access to the interactive login cluster. Shall we remove this sentence?